fdbserver/VFSAsync.cpp

```diff
@@ -223,14 +223,13 @@ static int VFSAsyncFileSize(sqlite3_file *pFile, sqlite_int64 *pSize){
 }
 
 static int asyncLock(sqlite3_file *pFile, int eLock){
-	VFSAsyncFile *p = (VFSAsyncFile*)pFile;
+	//VFSAsyncFile *p = (VFSAsyncFile*)pFile;
```

Comment:
This variable is used by the commented debug printf() statement immediately following its declaration.